python/lsst/ts/eas/diurnal_timer.py

```diff
@@ -231,6 +232,28 @@ async def stop(self) -> None:
         async with self.twilight_condition:
             self.twilight_condition.notify_all()
 
+    def sun_altitude_at(self, t_utc: float) -> float:
+        """Returns the sun altitude in degrees at the specified time.
+
+        Parameters
+        ----------
+        t_utc : float
+            The time of interest to look up sun altitude, specified
+            in UTC seconds in the UNIX epoch.
+
+        Returns
+        -------
+        float
+            Elevation above the local observatory horizon of the center of the
+            sun, in degrees.
+        """
+        return get_sun_altitude_deg(Time(t_utc, format="unix"))
+
+    @property
+    def seconds_until_twilight(self) -> float:
+        """Returns the number of seconds remaining until twilight."""
```

Comment:
That shall be method, with time as an argument. And shall check if the returned time is positive and less than 24 hours - if that's not the case, it shall throw an error. I understand twilight_time is set in async task that runs forever, but in case it wasn't started or isn't running, returned wrong value might cause problems.